simplesamlphp-module-privacyidea/templates/LoginForm.php

```diff
@@ -155,48 +155,45 @@ class="float-l erroricon" alt="gtk-dialog-error"/>
                                 <input id="password" name="password" tabindex="1" type="password" value="" class="text"
                                        placeholder="<?php echo htmlspecialchars($passHint, ENT_QUOTES) ?>"/>
 
-                                <strong id="message"><?php echo $this->data['message'] ?: "" ?></strong>
-
-                                <br><br>
-                                <label for="otp" class="sr-only">
-                                    <?php echo $this->t('{privacyidea:privacyidea:otp}'); ?>
-                                </label>
-
-                                <input id="otp" name="otp" tabindex="1" type="password" value="" class="text"
-                                       placeholder="<?php echo htmlspecialchars($otpHint, ENT_QUOTES) ?>"/>
+                                <strong id="message"><?php echo @$this->data['message'] ?: "" ?></strong>
```

Comment:
is HTML allowed in there? Otherwise this is a XSS vulnerability.

Reply:
Have to write HTML somewhere. How to mitigate? Or what would be an alternative approach?